packages/sui-test/bin/sui-test-e2e.js

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env node
+/* eslint no-console:0 */
+
+const path = require('path')
+const program = require('commander')
+const { getSpawnPromise, showError } = require('@s-ui/helpers/cli')
+
+const CYPRESS_FOLDER_PATH = path.resolve(__dirname, 'cypress')
+const TESTS_FOLDER = process.cwd() + '/test/e2e'
```

Comment:
We have to avoid run this tests in the others modes

Reply:
Will do

Reply:
@carlosvillu and @miduga it's harder that i thought. In sui-test-browser it's fine because it's a glob pattern. But in sui-test-server it's a folder that defaults to `test` : https://github.com/SUI-Components/sui/blob/master/packages/sui-test/bin/sui-test-server.js#L29

I see 2 options:
1. allowing a pattern in my command as it's in server.
2. allowing the ignorePattern option of cypress (it uses minimatch https://docs.cypress.io/guides/references/configuration.html#Global)

In both cases it's a the developer who manages it...
Do you have any better idea ?

Reply:
I think that the easier way could be put the e2e folder in the root project. That means make a clear diferencie  between them.

What do you think ?!

Reply:
Will do !